pf/tfgen/main.go

```diff
@@ -294,25 +285,3 @@ func mergeRenames(renames []tfgen.Renames) tfgen.Renames {
 	}
 	return main
 }
-
-func writeRenames(renames tfgen.Renames, opts tfgen.GeneratorOptions) error {
```

Comment:
So since renames are part of metadata now metadata writing to bridge-metadata.json already writes renames there. I think this method shouldn't be around as it can clobber bridge-metadata.json.

Reply:
I agree. This method was only for the pre-bridge based metadata world.